todoman/cli.py

```diff
@@ -80,6 +80,7 @@ def _sort_callback(ctx, param, val):
 
 
 def _todo_property_options(command):
+    click.option('--location', help=('The location of the todo.'))(command)
```

Comment:
Please change this to "The location where this todo takes place". The difference is subtle, but the current text might be misinterpreted as "the locaiton of the todo [file]".